internal/ethapi/transaction_args.go

```diff
@@ -122,16 +124,6 @@ func (args *TransactionArgs) setDefaults(ctx context.Context, b Backend) error {
 				if err != nil {
 					return err
 				}
-				if b.ChainConfig().IsEspresso(head.Number) {
```

Comment:
Above this price should probably be the suggested gas price, not to the suggested gas tip cap (thought to be low relative to the basefee)